### ISSUE TITLE
[Accessibility] Narrator doesn't announce control type for 'Calendar date picker' control.

### ISSUE THREAD

**Issue by almedina-ms** [microsoft/microsoft-ui-xaml#3317]
**Describe the bug**
Narrator doesn't announce control type for 'Calendar date picker' control. It only announces 'Calendar date picker' while user navigates on it. 

**Steps to reproduce the bug**
1. Create a blank app with a Calendar Date Picker control on it
2. Open narrator and set focus on the control created above

**Expected behavior**
Narrator should announce button as control type for 'Calendar date picker' control o that screen reader user could easily perform the functionality of such control.

**Version**
Microsoft.UI.Xaml 2.0.0
Microsoft.Toolkit.Uwp.UI.Controls 6.1.0

| Windows 10 version                  | Saw the problem? |
| :--------------------------------- | :-------------------- |
| Insider Build (200917-1700)      | Yes   |

**Additional context**
More details and videos in ADO bug 23702956

**Comment by StephenLPeters** [comment]
I think that the issue this is highlighting is that the default template for date picker is a button and when you navigate to it using narrator it doesn't announce a date picker until you expand it.

**Comment by YuliKl** [comment]
I recommend **not fixing** this issue because the control type "picker" is already being announced. 
Narrator announces a “calendar date picker”, a control to help users pick a calendar date. I believe the concept of pickers is actually pretty familiar to users while announcing something as a “picker button” would be confusing because these are two different control types. The interactions for pickers and buttons are somewhat similar (Space or Enter to activate), but only picker is expected to provide a supplemental UI to enable picking.

**Comment by StephenLPeters** [comment]
agreed
